api/HTMLVideoElement.json

```diff
@@ -781,16 +903,16 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLVideoElement/poster",
           "support": {
             "chrome": {
-              "version_added": "1"
+              "version_added": true
```

Comment:
Are these changes to the `poster` feature spurious? Seems like this is unrelated, but I'm not an expert. It's a bit surprising to see the switch from real version numbers here